renderer/components/ActivitiesTable.tsx

```diff
@@ -475,6 +477,211 @@ export default function ActivitiesTable({
       </tbody>
     </table>
   );
+
+  const compactView = (
+    <>
+      <h2
+        id="manual-input-title"
+        className="text-lg font-medium text-gray-900 dark:text-dark-heading"
+      >
+        Registrations
+      </h2>
+      <table className="min-w-full divide-y divide-gray-300 table-fixed dark:divide-gray-600">
+        <tbody className="text-gray-500 dark:text-slate-400">
+          {tableActivities.length > 0 &&
+            tableActivities?.map((activity, i) => (
+              <Fragment key={i}>
```

Comment:
It's nice to split this file for some small components and use it here. Because this file is too big and it's not testable at all. You could think about a bit more and try to create new components.